app/controllers/decks_controller.rb

```diff
@@ -1,13 +1,18 @@
 class DecksController < ApplicationController
 
   def index
-    @decks = Deck.all.order(:title)
+    @decks = Deck.all
+    @decks_ordered = @decks.order(:title)
+    @categories = Category.all
+    @category_deck_links = CategoryDeckLink.all
```

Comment:
we won't need this
https://github.com/eruditionapp/eruditionapp/pull/18/files#r57840712